src/packages/StorageFileApi.ts

```diff
@@ -788,6 +790,43 @@ export default class StorageFileApi {
     }
   }
 
+  /**
```

Comment:
listV2 naming is weird, but since this is an experimental method that will be the default in the next major version, approving it for now